src/components/UserProfile/UserProfile.jsx

```diff
@@ -1,4 +1,3 @@
-import React, { useState, useEffect, useRef } from 'react';
```

Comment:
Hey Alan, I checked your most recent commit, may be you deleted this line accidentally.
I checked functionalities with added this line back. Everything works fine.